cmd/restic/cmd_copy.go

```diff
@@ -0,0 +1,218 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/restic/restic/internal/debug"
+	"github.com/restic/restic/internal/errors"
+	"github.com/restic/restic/internal/restic"
+
+	"github.com/spf13/cobra"
+)
+
+var cmdCopy = &cobra.Command{
+	Use:   "copy [flags] [snapshotID ...]",
+	Short: "Copy snapshots from one repository to another",
+	Long: `
+The "copy" command copies one or more snapshots from one repository to another
+repository. Note that this will have to read(download) and write(upload) the
+entire snapshot(s) due to the different encryption keys on the source and
+destination.
+`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return runCopy(copyOptions, globalOptions, args)
+	},
+}
+
+// CopyOptions bundles all options for the copy command.
+type CopyOptions struct {
+	Repo            string
+	password        string
+	PasswordFile    string
+	PasswordCommand string
+	KeyHint         string
+	Hosts           []string
+	Tags            restic.TagLists
+	Paths           []string
+}
+
+var copyOptions CopyOptions
+
+func init() {
+	cmdRoot.AddCommand(cmdCopy)
+
+	f := cmdCopy.Flags()
+	f.StringVarP(&copyOptions.Repo, "repo2", "", os.Getenv("RESTIC_REPOSITORY2"), "repository to backup to or restore from (default: $RESTIC_REPOSITORY2)")
```

Comment:
Please change `repository to backup to or restore from` to `destination repository to copy snapshots to`.

Reply:
is changed.